test/Microsoft.AspNet.Mvc.FunctionalTests/MvcTagHelpersTest.cs

```diff
@@ -334,5 +339,41 @@ public class MvcTagHelpersTest
 Products: Laptops (3)";
             Assert.Equal(expected3, response4.Trim());
         }
+
+        [Theory]
+        [InlineData(null)]
+        [InlineData(true)]
+        [InlineData(false)]
+        public async Task FormTagHelper_GeneratesExpectedContent(bool? optionsAntiForgery)
+        {
+            // Arrange
+            var newServices = new ServiceCollection();
+            newServices.ConfigureTagHelpers().ConfigureForm(options => options.AntiForgery = optionsAntiForgery);
```

Comment:
please test these extension methods a bit more. for example not currently testing the `ConfigureForm(IConfiguration)` overload.  probably a unit test or three, not more functional testing.